modules/carto/src/style/color-bins-style.js

```diff
@@ -0,0 +1,18 @@
+import {scaleThreshold} from 'd3-scale';
+import {getPalette, NULL_COLOR} from './utils';
+
+export default function colorBins({breaks, colors, nullColor = NULL_COLOR}) {
+  if (Array.isArray(breaks)) {
```

Comment:
I don't understand this if. In which case breaks it's not an array?  Cannot we just remove this if?

Reply:
Let's say the user incorrectly passes `breaks` as other type different of `Array`, we are not actually handling errors in that case, just returning `NULL_COLOR`

Reply:
I would also propose `if (Array.isArray(breaks) && breaks.every(Number.isFinite))`